Orange/widgets/utils/plot/owplotgui.py

```diff
@@ -365,20 +377,53 @@ def _check_box(self, widget, value, label, cb_name):
             When the checkbox is toggled, the attribute ``value`` of the plot object is set to the checkbox' check state,
             and the callback ``cb_name`` is called.
         '''
-        gui.checkBox(widget, self._plot, value, label, callback=self._get_callback(cb_name))
+        return gui.checkBox(widget, self._plot, value, label, callback=self._get_callback(cb_name))
 
     def antialiasing_check_box(self, widget):
         '''
             Creates a check box that toggles the Antialiasing of the plot
         '''
         self._check_box(widget, 'use_antialiasing', 'Use antialiasing', 'update_antialiasing')
 
+    def jitter_size_slider(self, widget):
+        values = self._plot.master.jitter_sizes if hasattr(self._plot.master, "jitter_sizes") \
```

Comment:
This could be `values = getattr(self._plot.master, "jitter_sizes", self.JITTER_SIZES)`. 

Is it not `"JITTER_SIZES"`, not `"jitter_sizes"`? I suppose https://github.com/biolab/orange3/pull/2541/commits/49cf76d6006d060f170315e5dff06b41aa9dee97#diff-d6ec84862acc9707da06d662d6752cbdR489 should define this sizes, right?

However, I'd prefer passing jitter sizes as an argument to this function instead of forcing the widget to specify it in a completely different location with a prescribed name. E.g.
`def jitter_size_slider(self, widget, jitter_sizes=JITTER_SIZES):`. (For this to work without warnings, `JITTER_SIZES` would have to be a tuple, not a list.)